lectures/global-entrans-tuwien/index.rst

```diff
@@ -5,27 +5,26 @@ Global Energy Transitions and Climate Policy @ TU Wien
    :width: 240px
    :align: right
 
-This course is part of the Master's Degree Program in `Environmental Technology & International Affairs`_ 
-held at the Technical University Vienna (TU Wien) since spring 2021.
+This course is part of the Master's degree program in `Environmental Technology & International Affairs`_ 
+held at the Technical University Vienna (TU Wien).
 The course is designed and delivered by `Dr. Behnam Zakeri <https://www.iiasa.ac.at/staff/zakeri>`_,
-including two lectures given by guest speakers Dr. Kavita Surana and Franziska Menten (MSc).
+including a few lectures by guest speakers/lecturers.
```

Comment:
```suggestion
including several lectures by guest speakers/lecturers.
```

Reply:
Thanks @danielhuppmann for the quick feedback. I corrected this.